src/averageAges.js

```diff
@@ -55,10 +77,25 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // 1. find a mother of each person (or only for men)
-  // 2. keep people who have mothers in the array
-  // 3. calculate the difference child.born - mother.born
-  // 4. return the average value
+  const ageDiffs = [];
+
+  people.forEach(mother => {
```

Comment:
Replace the 'forEach' loop with more suitable array methods like 'filter', 'map', 'some', 'every', 'find', 'reduce'.